terraform/wait-windows-host.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# we don't want to exit when send-command fails
+set +e
+
+instance_id=$1
+
+while true
+do
+    instance_status=$(aws ec2 describe-instance-status --instance-ids $instance_id --no-cli-pager --query 'InstanceStatuses[0].InstanceStatus.Status' --output text)
```

Comment:
if there are 2 builds then this assumes index 0, do we protect for this any way? should we at least say something in the action description?